botx/models/messages/sending/message.py

```diff
@@ -24,13 +24,14 @@
     "{0} can not be passed along with manual validated_values for it"
 )
 
-EMBED_MENTION_TEMPATE = (
+EMBED_MENTION_TEMPLATE = (
     "<embed_mention:{mention_type}:"
     "{mentioned_entity_id}:{mention_id}:{mention_name}>"  # noqa: WPS326
 )
 EMBED_MENTION_RE: Final = re.compile(
-    "<embed_mention:(?P<mention_type>.+?):(?P<mentioned_entity_id>.+?)"
-    ":(?P<mention_id>.+?):(?P<name>.+?)?>",  # noqa: WPS326 C812
+    "<embed_mention:(?P<mention_type>.+?):"
```

Comment:
```suggestion
    "<embed_mention:(?P<mention_type>user|contact|chat|channel|all):"
```

Reply:
I think it's overspecification. This check already [exists](https://github.com/ExpressApp/pybotx/blob/71eedabfb10a88de82a3b62ac669315b9e25c9dd/botx/models/messages/sending/message.py#L528).

Reply:
Ok. So let's add unit-test, that checks wrong mention type will raise an exception.